.github/workflows/ci.yml

```diff
@@ -28,4 +28,4 @@ jobs:
       - name: Docker Login
         run: docker login --username "$DOCKERHUB_USERNAME" --password "$DOCKERHUB_TOKEN"
       - name: Run build
-        run: earthly --org applied-knowledge-systems --sat my-satellite --ci --push +pipeline
\ No newline at end of file
+        run: earthly --org applied-knowledge-systems --sat bigbox-sattelite --ci --push +pipeline
```

Comment:
There's a type on `sattelite`. It might still be correct, if that is the name of the satellite, which was configured. Just fyi. 😉